Silicon/CommonSocPkg/Include/GpioLibConfig.h

```diff
@@ -0,0 +1,506 @@
+/** @file
+  Header file for GpioConfig structure used by GPIO library.
+
+  Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include <Library/GpioInfoLib.h>

```

Comment:
Does this library depend on GpioInfoLib.h or GpioInfoLib.h depend on this library?
Should not include it here if this library does not depend on GpioInfoLib.h.

Reply:
True, this is redundant. Removed it.